tmt/options.py

```diff
@@ -250,115 +279,141 @@ def option(
     *FILTER_OPTION_LONG,
     option(
-        '--condition', 'conditions', metavar="EXPR", multiple=True,
-        help="Use arbitrary Python expression for filtering."),
+        '--condition',
+        'conditions',
+        metavar="EXPR",
+        multiple=True,
+        help="Use arbitrary Python expression for filtering.",
+    ),
     option(
-        '--enabled', is_flag=True,
-        help="Show only enabled tests, plans or stories."),
+        '--enabled',
+        is_flag=True,
+        help="Show only enabled tests, plans or stories.",
+    ),
     option(
-        '--disabled', is_flag=True,
-        help="Show only disabled tests, plans or stories."),
+        '--disabled',
+        is_flag=True,
+        help="Show only disabled tests, plans or stories.",
+    ),
     option(
-        '--link', 'links', metavar="RELATION:TARGET", multiple=True,
+        '--link',
+        'links',
+        metavar="RELATION:TARGET",
+        multiple=True,
         help="""
              Filter by linked objects (regular expressions are supported for both relation and
              target).
-             """),
+             """,
+    ),
     option(
-        '--exclude', 'exclude', metavar='[REGEXP]', multiple=True,
-        help="Exclude a regular expression from search result."),
-    ]
+        '--exclude',
+        'exclude',
+        metavar='[REGEXP]',
+        multiple=True,
+        help="Exclude a regular expression from search result.",
+    ),
+]
 
 
 STORY_FLAGS_FILTER_OPTIONS: list[ClickOptionDecoratorType] = [
-    option(
-        '--implemented', is_flag=True,
-        help='Implemented stories only.'),
-    option(
-        '--unimplemented', is_flag=True,
-        help='Unimplemented stories only.'),
-    option(
-        '--verified', is_flag=True,
-        help='Stories verified by tests.'),
-    option(
-        '--unverified', is_flag=True,
-        help='Stories not verified by tests.'),
-    option(
-        '--documented', is_flag=True,
-        help='Documented stories only.'),
-    option(
-        '--undocumented', is_flag=True,
-        help='Undocumented stories only.'),
-    option(
-        '--covered', is_flag=True,
-        help='Covered stories only.'),
-    option(
-        '--uncovered', is_flag=True,
-        help='Uncovered stories only.'),
-    ]
+    option('--implemented', is_flag=True, help='Implemented stories only.'),
+    option('--unimplemented', is_flag=True, help='Unimplemented stories only.'),
+    option('--verified', is_flag=True, help='Stories verified by tests.'),
+    option('--unverified', is_flag=True, help='Stories not verified by tests.'),
+    option('--documented', is_flag=True, help='Documented stories only.'),
+    option('--undocumented', is_flag=True, help='Undocumented stories only.'),
+    option('--covered', is_flag=True, help='Covered stories only.'),
+    option('--uncovered', is_flag=True, help='Uncovered stories only.'),
```

Comment:
Let's force comma-separate this for consistency with above

Reply:
Fixed in fa6aedff.